content/docs/user-guide/what-is-dvc.md

```diff
@@ -1,16 +1,18 @@
 # What Is DVC?
 
-**Data Version Control** is a new type of data versioning, workflow and
-experiment management software, that builds upon [Git](https://git-scm.com/)
-(although it can work stand-alone). DVC reduces the gap between established
-engineering tool sets and data science needs, allowing users to take advantage
-of new [features](#core-features) while reusing existing skills and intuition.
+**Data Version Control** is a new type of data versioning, workflow and machine
+learning experiment management software that builds upon
```

Comment:
> machine learning experiment management

Makes sense but doesn't even register as a term (https://trends.google.com/trends/explore?geo=US&q=machine%20learning%20experiment%20management,experiment%20management). Then again "experiment management software" doesn't register either... So not sure it's a problem. But is there another way to insert "machine learning" in this paragraph without making it hard to read? Or maybe just use "ML" here and insert the full term elsewhere in the p/doc?

Reply:
Was after specific term "[version machine learning experiments](https://www.google.com/search?client=safari&rls=en&q=version+machine+learning+experiments&ie=UTF-8&oe=UTF-8)", which is awkward to insert. I like the idea to use "ML" here... there's already a lot going on in that sentence. 

In a lot of cases a term might appear in search console or related terms, and I'm looking to expand where DVC.org has impressions but low/zero clicks. Many are domain-specific and very low volume (relative to Internet scale) so they might not appear in Google trends at all.

Reply:
> Was after specific term "version machine learning experiments"

I see. Let's find another place to add the term in this doc? Feel free to add sentences, paragraphs, or even sections if needed.

Reply:
OK, added this in a rewrite of the other paragraph.